modules/logger/logger.py

```diff
@@ -172,3 +173,25 @@ def critical(self, message: str, log_with_frame_info: bool = True) -> None:
             caller_frame = logger_frame.f_back
             message = self.message_and_metadata(message, caller_frame)
         self.logger.critical(message)
+
+    def save_image(
+        self, image: np.ndarray, filename: str, log_with_frame_info: bool = True
```

Comment:
Can you add an optional parameter: log_info_message with default to false, and only log a message if this is true?

Reply:
would log_with_frame_info's part stay the same? I'd only had an if condition for line 199 correct?